docs/cloud/tcloud_getting_started.md

```diff
@@ -64,9 +64,13 @@ Technical Requirements:
 
 The first step to setting up Tobiko Cloud is logging in to the web interface:
 
-1. Open a browser and navigate to the Tobiko Cloud URL (ex: https://cloud.tobikodata.com/sqlmesh/tobiko/public-demo/observer/)
-2. Leave the username blank and use the temporary password you received from the Solutions Architect in the temporary password link
-3. Once logged in, you should see the home page. Your view should be empty, but the figure below shows a populated example with Tobiko Cloud running in production:
+1. We will authenticate into your Tobiko Cloud instance. If it is your first time going through this flow, your Solutions Architect will    guide you on how to get SSO configured. Open the url below.
```

Comment:
```suggestion
1. We will authenticate into your Tobiko Cloud instance. If it is your first time going through this flow, your Solutions Architect will guide you on how to get SSO configured. Open the url below.
```

Reply:
how to get SSO configured

add hyperlink to the SSO docs so people can skim beforehand easily.